Gemfile

```diff
@@ -28,22 +28,14 @@ group :system_tests do
   gem 'beaker-puppet_install_helper', :require => false
 end
 
-is_x64 = Gem::Platform.local.cpu == 'x64'
-if is_x64
-  platform(:x64_mingw) do
-    gem "win32-dir", "~> 0.4.9", :require => false
-    gem "win32-process", "~> 0.7.4", :require => false
-    gem "win32-service", "~> 0.8.4", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-  end
-else
-  platform(:mingw) do
-    gem "win32-process", "~> 0.6.5", :require => false
-    gem "win32-service", "~> 0.7.2", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-    if RUBY_VERSION =~ /^1\./ then
-      gem "win32console", :require => false
-    end
+platforms :mswin, :mingw do
```

Comment:
> The problem is that :mswin doesn't actually match our x64 Ruby runtime, and won't install platform specific gems in an x64-mingw32 environment. See bundler code for how the platforms are determined - it can be surprising. (As you've noted, you're seeing the desired end result, but it's a bit of a coincide that it works - the :mswin part is meaningless)

Not really a coincidence based on what I laid out in my explanation. But as I mentioned, making it more obvious may be what is necessary to show why it is not a coincidence.